c/meterpreter/source/extensions/priv/service.h

```diff
@@ -9,4 +9,6 @@ DWORD service_create( char * cpName, char * cpPath );
 
 DWORD service_destroy( char * cpName );
 
+DWORD query_service_status( char * cpName, DWORD* dwState );
```

Comment:
To fit in with the others here, this should be named `service_query_status` so it's `service_$verb`